client/js/lib/prosemirror/converter.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * This ProseMirrorConverter class is responsible for converting
+ * ProseMirror data to an HTML string and to convert proseMirror data
+ * to an HTML string.
+ *
+ * Example usage:
+ * - from ProseMirror to html: `converter.fromProseMirror(ProseMirrorData).toHtml().getHtml()`
+ * - from html to ProseMirror: `converter.fromHtml(htmlString).toProseMirror()`
+ *
+ * @param proseMirrorData - object
+ * @param htmlString - string
+ */
+export class ProseMirrorConverter {
+  constructor() {
+    this.prosemirrorData = null
+    this.htmlString = null
+  }
+
+  /**
+   * Returns a ProseMirrorConverter instance filled with ProseMirror data.
```

Comment:
Nitpick: At this point it is not perfectly clear what "ProseMirror data" may be.

Reply:
I see, I tried to be more explicit here: https://github.com/demos-europe/demosplan-core/pull/3875/commits/4ba7fe8b305eb22aa506d0e9ae95102779fa1faa and hope by adding a documentation link too, things will be more clear.

Reply:
thanks! i think this makes it clear for "first time readers" of that code.